app/views/homepage/index.html.erb

```diff
@@ -115,7 +127,21 @@
 
         <div class="govuk-grid-column-one-third">
           <a href="/government/topical-events/budget-2020" aria-hidden="true" tabindex="-1">
-            <%= image_tag 'homepage/budget-2020.jpg', alt: '', class: 'home-promo__image', loading: "lazy" %>
+            <%= image_tag "homepage/budget-2020--480w.jpg", {
+              alt: "",
+              class: "home-promo__image",
+              height: 400,
+              loading: "lazy",
+              sizes: "(max-width: 640px) 100vw, (max-width: 960px) 33vw, 300px",
```

Comment:
This could potentially be 1020px rather than 960px, as the 30px of margin either side means we continue to take up 33vw of the screen until we reach 1020px. But I don't think it'll make a difference in practice.

```suggestion
              sizes: "(max-width: 640px) 100vw, (max-width: 1020px) 33vw, 300px",
```